.github/workflows/check_code.yml

```
@@ -0,0 +1,48 @@
+
+      # Запуск Black (форматирование)
+      - name: Run Black
+        run: black --check .
+
+      # Запуск Ruff (линтинг)
+      - name: Run Ruff
+        run: ruff .
+
+      # Запуск Pylint (проверка качества кода)
+      - name: Run Pylint
+        run: pylint $(git ls-files '*.py')
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:1:8: Quote this to prevent word splitting

(shellcheck)